docs/antctl.md

```diff
@@ -0,0 +1,66 @@
+# Antctl
```

Comment:
Could we make a separate PR or at least a separate commit for the documentation?

Reply:
OK, I removed this markdown from this CL. It will be submitted separately in another CL.